freqtrade/optimize/edge.py

```diff
@@ -0,0 +1,106 @@
+# pragma pylint: disable=missing-docstring, W0212, too-many-arguments
```

Comment:
Please call this file `edge_cli.py` (or something else)
having a file "edge" (which doesn't contain the class `Edge`) - and a module "edge" is asking for problems - when importing you need to be very carefull which will be imported